ui/src/admin/apis/index.js

```diff
@@ -61,3 +61,31 @@ export const deleteUser = async (url, addFlashMessage, username) => {
     })
   }
 }
+
+export const updateRoleUsers = async (url, users) => {
```

Comment:
I was considering that.  Ultimately, I didn't want to do logic around what to do if a particular property wasn't present.  Maybe I'm missing something?